virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,40 @@
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+By default, {VirtProductName} is installed with a single, internal pod network.
 
-To attach a virtual machine to an additional network:
+You must create a Linux bridge network attachment definition (NAD) in order to connect to external networks. Otherwise, your network interface cards will be unavailable.
 
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
-
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
-
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+To attach a virtual machine to an external network:
 
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
+. Create a `NodeNetworkConfigurationPolicy`.
+. Create a Linux bridge NAD.
+. Configure the virtual machine, enabling the virtual machine to recognize the NAD.
 
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+1]
 
-For more information about scheduling, interface types, and other node networking activities,
-see the xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking]
-section.
-
-[id="virt-creating-network-attachment-definition"]
-== Creating a network attachment definition
+[id="virt-creating-linux-bridge-network-attachment-definition"]
+= Creating a Linux bridge network attachment definition
 
 [WARNING]
 ====
-Configuring ipam in a network attachment definition for virtual machines is not supported.
+Configuring IPAM in a network attachment definition for virtual machines is not supported.
 ====
 
 include::modules/virt-creating-bridge-nad-web.adoc[leveloffset=+2]
 
 include::modules/virt-creating-bridge-nad-cli.adoc[leveloffset=+2]
 
-[id="virt-attaching-vm-to-additional-network"]
-== Attaching the virtual machine to the additional network
+[id="virt-configuring-vm-for-additional-network"]
+= Configuring the virtual machine for an additional network
```

Comment:
I thought of a title that might be better. "Configuring the Linux bridge interface [of? for? in?] the virtual machine"

Reply:
We just changed this to _Configuring a virtual machine network interface to use the default network_ but I do prefer Configuring a virtual machine for a Linux bridge network ---- that seems to be the major use case. As you say above, why are configuring the config that comes with the product by default (there are other steps but to me, it seems that accessing external networks is the most common task.

Reply:
Changed this title again to: Configuring a virtual machine for a Linux bridge network

I agree that this is more succinct and clear.